### PR TITLE
override addChid of UIKit

### PR DESCRIPTION
**Pull request by kazuki-horie-m**
There is too many func of addXXXChild.
so override default addChild.

### CODE REVIEW

File: RxController/Classes/RxViewController.swift [hunk 1]
```diff
@@ -27,12 +27,24 @@ import UIKit
 import RxSwift
 import RxCocoa
 
-open class RxViewController<ViewModel: RxViewModel>: UIViewController {
+protocol RxViewControllerProtocol {
+    var view: UIView! { get }
```
Comment (lm2343635) [inline RxController/Classes/RxViewController.swift]: Is IUO (`UIView!`) necessary here?

File: RxController/Classes/RxViewController.swift [hunk 2]
```diff
@@ -73,58 +85,44 @@ open class RxViewController<ViewModel: RxViewModel>: UIViewController {
...
+        viewModel.addChildModel(childController.rxViewModel)
+
+        // Set the parent events property of the child view model.
+        childController.rxViewModel._parentEvents = viewModel.events
+
+        // Add child view controller to the parent view controller.
+        containerView.addSubview(childController.view)
+        childController.didMove(toParent: self)
+
+        childController.view.translatesAutoresizingMaskIntoConstraints = false
```
Comment (lm2343635) [inline RxController/Classes/RxViewController.swift]: The constraints creator should be always executed.

Comment (lm2343635) [reply RxController/Classes/RxViewController.swift]: If a controller is not a RxViewController, the constraints create will be not executed.

File: RxController/Classes/RxViewController.swift (hunk 1 above)
Comment (kazuki-horie-m) [reply RxController/Classes/RxViewController.swift]: yes.
using in `addChild(_ childController: UIViewController, to containerView: UIView)`

**Pull request merged**